src/components/Article.jsx

```diff
@@ -95,6 +94,15 @@ console.log(user)
     }
 
 
+    function handleDelete(id) {
+        deleteComment(id).then(() => {
+            alert('Comment Deleted!')
+        }).catch((err) => {
+            if(err){
+                alert('Error deleting comment! Please try again')
+            }
```

Comment:
This message doesn't come up when there's an error. Try deleting the catch block in your utils file and test it with 'offline' mode.